src/components/PaymentPlan/PaymentPlanCard.jsx

```diff
@@ -23,7 +22,6 @@ const PaymentPlanCard = ({
     ...otherProps
 }) => {
     const { isOpen, onOpen, onClose } = useDisclosure();
-    const boxColor = useColorModeValue('brand.100', 'brand.700');
     const PaymentModal = ({ payableAmount }) => {
         return (
```

Comment:
I think this function just has a return function 
Can be removed and replaced with ()